component-models.json

```diff
@@ -245,6 +245,29 @@
       }
     ]
   },
+  {
+    "id": "cards",
+    "fields": [
+      {
+        "component": "select",
+        "name": "classes",
+        "label": "Card View Layout",
+        "description": "Profile View Layout to show cards as flex, and each card to have image on left and text on right, default view grid",
+        "valueType": "string",
+        "value": "",
+        "options": [
+          {
+            "name": "Default View",
+            "value": ""
+          },
+          {
+            "name": "Hero - Profile View",
+            "value": "hero"
```

Comment:
change it to "Profile - Summary view", value: "profile-summary"

Reply:
Done